### PR TITLE
feat [#W3.3] : front-end

### PR DESCRIPTION
**Pull request by Aleksa1998**

### CODE REVIEW

File: PatientWebApplication/HealthClinic.DAL/Service/RegularAppointmentService.cs [hunk 1]
```diff
@@ -445,11 +445,16 @@ private List<DoctorAppointment> SearchForRoomAdvanced(List<DoctorAppointment> ap
...
         /// <returns> List of valid appointments. </returns>
-        public List<DoctorAppointment> FindAllValidAppointments(List<DoctorAppointment> allValidAppointments, List<Survey> surveys)
+        public List<DoctorAppointment> FindAllValidAppointmentsWithoutSurvey(List<DoctorAppointment> allValidAppointments, List<Survey> surveys)
         {
             return CheckIfAppointmentsHappened(allValidAppointments.Where(p => !FindAllUnvalidAppointments(surveys).Any(p2 => p2 == p.id)).ToList());
         }
 
+        public List<DoctorAppointment> FindAllValidAppointmentsWithSurvey(List<DoctorAppointment> allValidAppointments, List<Survey> surveys)
+        {
+            return CheckIfAppointmentsHappened(allValidAppointments.Where(p => FindAllUnvalidAppointments(surveys).Any(p2 => p2 == p.id)).ToList());
```
Comment (trki293) [inline PatientWebApplication/HealthClinic.DAL/Service/RegularAppointmentService.cs:455]: The variables p and p2 need to be renamed to make more sense.

**Review by trki293 — CHANGES_REQUESTED** [review]
Modify the indicated place in the code.

**Review by trki293 — APPROVED** [review]
PR is approved.

**Review by Nemanja0505 — APPROVED** [review]
PR is approved. Good Clean code principles were implemented.

**Pull request merged**